### PR TITLE
[v6r13] Pylint and fixes v6r13 - 2

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: ConfigurationSystem/Client/CSAPI.py [hunk 1]
```diff
@@ -411,12 +411,12 @@ def addShifter( self, shifters = None ):
     :return: S_OK/S_ERROR
     """
 
-    def getOpsSection( ):
+    def getOpsSection():
```
Comment (chaen) [inline ConfigurationSystem/Client/CSAPI.py]: Without 'self' or 'cls' ?

Comment (fstagni) [inline ConfigurationSystem/Client/CSAPI.py]: This is method inside a method... :)

Comment (chaen) [inline ConfigurationSystem/Client/CSAPI.py]: Oh sorry, missed that :-)

Comment (atsareg) [inline ConfigurationSystem/Client/CSAPI.py]: This is a method which is raising Exception contrary to DIRAC conventions. Should return S_ERROR() instead

Comment (sposs) [inline ConfigurationSystem/Client/CSAPI.py]: Actually, I disagree... In this case, the exception is perfectly suitable. If you look where the try/except clause is, it makes perfect sense to use that.

Comment (atsareg) [inline ConfigurationSystem/Client/CSAPI.py]: Here using S_ERROR or exception is pretty much equivalent and will take the same amount of code. Absolutely no advantage of using exception. But we should be consistently using conventions of the project. Uniform style does matter in a multi-developer project. We discussed this zillions of times and let's not restart it.

**Pull request merged**